src/cascadia/TerminalApp/ShortcutActionDispatch.cpp

```diff
@@ -58,6 +59,8 @@ namespace winrt::TerminalApp::implementation
                 "ActionDispatched",
                 TraceLoggingDescription("Event emitted when an action was successfully performed"),
                 TraceLoggingValue(static_cast<int>(actionAndArgs.Action()), "Action"),
+                TraceLoggingValue(GetBranding(), "Branding"),
+                TraceLoggingValue(IsPackaged(), "IsPackaged"),

```

Comment:
I'd prefer something that tells us whether we're packaged, unpackaged or _portable_ if we have it